### PR TITLE
Added works section and about me.

### PR DESCRIPTION
**Pull request by alex1779**
Modifications on index.html and style.css. Added works section and about me. Added images.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -123,12 +120,357 @@ <h1 class="primary-text-1">ľm Mariane
...
+
 
     <div id="header-background-mobile">
     </div>
 
 
 
+
+    <!-- Works section -->
+    <div id="works-section">
```
Comment (omarsalem7) [inline index.html]: - [x] Kindly replace div with section to improve SEO for your portfolio.

File: index.html [hunk 2]
```diff
@@ -123,12 +120,357 @@ <h1 class="primary-text-1">ľm Mariane
...
+
+      </div>
+
+    </div>
+
+    <!-- about-myself -->
+    <div id="about-myself-base">
+    </div>
+
+    <div id="about-myself">
```
Comment (omarsalem7) [inline index.html]: - [x] Also here use section tag instead of div.

File: index.html [hunk 3]
```diff
@@ -123,12 +120,357 @@ <h1 class="primary-text-1">ľm Mariane
...
+            <div id="block-apart-tags">
+              <ul id="block-apart-tags-li">
+                <li id="block-apart-tags-item-1" class="block-apart-tags"><a class="block-apart-tags-item">html</a></li>
+                <li id="block-apart-tags-item-2" class="block-apart-tags"><a class="block-apart-tags-item">css</a></li>
+                <li id="block-apart-tags-item-3" class="block-apart-tags"><a class="block-apart-tags-item">javaScript</a></li>
+              </ul>
+            </div>
+
+            <div id="block-apart-action">
+              <button id="block-apart-action-button" type="button">See Project</button>
```
Comment (omarsalem7) [inline index.html:170]: - [x] Consider making sure the buttons follow the button styling: Active, Hover, and Pressed.

File: index.html [hunk 4]
```diff
@@ -123,12 +120,357 @@ <h1 class="primary-text-1">ľm Mariane
...
+          <h1 class="primary-text-1">About<br>
+             Myself</h1>
+        </div>
+
+        <div id="about-myself-top-primary-text-2">
+          <p class="primary-text-2">Hello ľm  a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </div>
+
+        <div id="about-myself-top-social-media">
+          <a class="social-media-p">LET'S CONNECT</a>
```
Comment (omarsalem7) [inline index.html]: - [x] It will be better if you use `hx` instead of `a` to improve SEO.

File: index.html [hunk 5]
```diff
@@ -123,12 +120,357 @@ <h1 class="primary-text-1">ľm Mariane
...
+              <div id="about-myself-skill-list-icon-3">
+              </div>
+
+              <a id="about-myself-skill-list-text-3" class="about-myself-skill-list-text">CSS</a>
+
+            </div>
+
+          </div>
+
+        </div>
```
Comment (omarsalem7) [inline index.html]: - [x] Kindly make sure the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @alex1779 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- There is no linter error present ✔️
- The Pull Request has a proper title and description. ✔️

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — APPROVED** [review]
#   *Status: Approved* :tada: :heavy_check_mark: 

Hi @alex1779, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

-  No linter errors. :heavy_check_mark: 
-  Good README. :heavy_check_mark: 
-  Good commit messages. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**